### PR TITLE
added resource for maximum_retry_attempts for the lambda

### PR DESCRIPTION
**Pull request by banovicn**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -109,3 +109,9 @@ variable "tags" {
   type        = map(string)
   description = "A mapping of tags to assign to the bucket"
 }
+
+variable "retries" {
+  type        = number
+  default     = null
+  description = "Number of maximum retries for the lambda"
```
Comment (svanharmelen) [inline variables.tf:116]:
```suggestion
  description = "Maximum number of retries for the Lambda invocation"
```

File: main.tf [hunk 2]
```diff
@@ -84,6 +84,12 @@ data "archive_file" "dummy" {
   }
 }
 
+resource "aws_lambda_function_event_invoke_config" "default" {
+  count                  = var.retries != null ? 1 : 0
+  function_name          = var.name
```
[the remark this answers is not in the document]
Comment (banovicn) [reply main.tf:89]: done

File: variables.tf [hunk 3]
```diff
@@ -109,3 +109,9 @@ variable "tags" {
   type        = map(string)
   description = "A mapping of tags to assign to the bucket"
 }
+
+variable "retries" {
```
[the remark this answers is not in the document]
Comment (banovicn) [reply variables.tf:113]: done

File: variables.tf (hunk 1 above)
Comment (banovicn) [reply variables.tf:116]: done

**Pull request merged**